public/app/percona/dbaas/components/DBCluster/EditDBClusterPage/hooks/useEditDBClusterFormSubmit.ts

```diff
@@ -8,13 +8,14 @@ import { ClusterSubmit, DBClusterFormSubmitProps } from '../EditDBClusterPage.ty
 export const useEditDBClusterFormSubmit = ({
   mode,
   showPMMAddressWarning,
+  settings,
 }: DBClusterFormSubmitProps): [ClusterSubmit, boolean | undefined, string, 'error' | 'ok' | undefined] => {
   const dispatch = useDispatch();
   const { result, loading } = useSelector(mode === 'create' ? getAddDbCluster : getUpdateDbCluster);
   const { selectedDBCluster } = useSelector(getDBaaS);
 
-  const addCluster = async (values: Record<string, void>) => {
-    await dispatch(addDbClusterAction({ values, setPMMAddress: showPMMAddressWarning }));
+  const addCluster = async (values: Record<string, any>) => {
```

Comment:
Is the `any` necessary here?

Reply:
yes, it's type from react-final-form, I tried to add myType to values (this also applies to the previous question) but I couldn't do it like `const var: MyType = something`, because the typescript asks to use Record<string, any> becouse of final-forms default type
If i tried to add my type it returns this error for each form
![image](https://user-images.githubusercontent.com/90199600/216947197-c5b96be0-d4fd-4dff-8198-7144a584ddf4.png)